docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+ Correct COMINtcvital path; Turn off SYNDATA in prepbufr
+
+SCRIPTS file changes
+--------------------
+ scripts/exglobal_dump.sh
+ scripts/exrap_dump.sh
+ scripts/exurma_dump.sh
+ scripts/exrtma_dump.sh
+ scripts/excdas_dump.sh
+ Add dump groups: gmi1cr, snocvr, saldrn, subpfl
+ Add Sentinel-6 data to gpsro (via dev to prod tank update)
+ Add Metop-B/C AMVs data from tank b005/xx081 to satwnd dump group
+
+ scripts/exglobal_makeprepbufr.sh
+ Add subpfl and saldrn to nsst bufr dump
+
+ scripts/exnam_makeprepbufr.sh
+ Comment out COMSPtcvital path to turn off SYNDAT for nam
+
+USH script changes
+------------------
+ ush/build.sh
+ Move build.sh from obsolete /build-obsproc to /ush
+
+DOCS file changes
+-----------------
+ docs/README.md
+ Update build instructions  
+
+VERSIONS file changes
+----------------
+ versions/run.ver
+ Secure turn off SYNDAT for nam
+
+FIX code changes
+-------------------
+ No changes.
+
+Source code changes
+-------------------
+ No changes
+
+Output changes
+--------------
+ Generate new dumps:
+ *gmi1cr*, *snocvr*, *saldrn*, *subpfl*
+ Update dumps:
+ *gpsro*, *satwnd* and *nsstbufr (*nsstbufr is generatd only in global) 
+
+Compute Resource Information
+----------------------------
+ Memory usage was increased, from 60 to 120gb, when testing with triggers/jglobal_dump.wc2.pbs
+
+Disk Space Changes 
+------------------
+ The newly generated dumps ( *gmi1cr*, *snocvr*, *saldrn*, *subpfl* ) vary in size from 1Mb to 200Mb
+ depending on network, cycle time, time dump window. 0 size dumps are possible
+ The updated dumps (*gpsro*, *satwnd* and *nsstbufr) are up to ~25% larger in size
+ For examples, see: /lfs/h2/emc/stmp/iliana.genkova/CRON/RELv1.1.0/com
+ 
+Shared Software 
+---------------
+ obsproc needs modules bufr-dump and prepobs in order to run
+
+Libraries
+---------
+ See
+ versions/build.ver 
+ versions/run.ver 
+   
+Build Instructions
+------------------
+ See README.md
+
+Preimplementation Testing Requirements
+--------------------------------------
+ Run jobs/JOBSPROC_networkname_DUMP and jobs/JOBSPROC_networkname_DUMP_POST
+ for the following networkname values: GLOBAL, RAP, URMA, RTMA, CDAS
+
+Dissemination
+-------------
+ No change in dissemination. 
+
+Special Instructions
+--------------------
+ 1.Implement release/obsproc.v1.1.0 and release/bufr-dump.v1.1.0 at the same time
+ 2.Implement release/obsproc.v1.1.0 and release/bufr-dump.v1.1.0 before or at the same time as GFS v16.3 
+
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+obsproc v1.0.0        -> released to NCO on Dec 21,2021
+        v1.0.2        -> implemented Jun 28, 2022             
+
+Transition from WCOSS Cray and Phase3 to WCOSS2
+======================================
+https://docs.google.com/document/d/1U4pu4iSlLOnbNrcBqBlOU7bnNcTFfe_MpjV8uKN4S40/edit
+
+Where is the release tag on subversion/git/vlab? 
+ git clone https://github.com/NOAA-EMC/obsproc
+ git checkout release/obsproc.v1.0.0
```

Comment:
The fact that the user has already checked out this tag, means they already know this.
Also not release note material.

Reply:
This is an older part of RN, which was already released, so I'll keep it for the records